crates/ruff/src/fs.rs

```diff
@@ -82,3 +84,10 @@ pub fn relativize_path_to<P: AsRef<Path>, R: AsRef<Path>>(path: P, project_root:
             .display()
     )
 }
+
+/// Read a string from `stdin`.
+pub fn read_from_stdin() -> Result<String, io::Error> {
```

Comment:
Can we move this to `ruff_cli`?

Reply:
I just realized we have the same method in the formatter too.